airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -241,6 +247,23 @@ public List<TableInfo<CommonField<JDBCType>>> discoverInternal(final JdbcDatabas
     }
   }
 
+  @Override
+  protected boolean verifyCursorColumnValues(final JdbcDatabase database, final String schema, final String tableName, final String columnName)
+      throws SQLException {
+    final String resultColName = "nullValue";
+
+    final String query = String.format(NULL_CURSOR_VALUE_WITH_SCHEMA_QUERY,
+        resultColName, schema, tableName, columnName, schema, tableName, columnName);
+
+    LOGGER.debug("null value query: {}", query);
+    final List<JsonNode> jsonNodes = database.bufferedResultSetQuery(conn -> conn.createStatement().executeQuery(query),
+        resultSet -> JdbcUtils.getDefaultSourceOperations().rowToJson(resultSet));
+    Preconditions.checkState(jsonNodes.size() == 1);
+    final boolean nullValExist = jsonNodes.get(0).get(resultColName).booleanValue();
```

Comment:
I had this weird thing here that column name was in lowercase. 
Not sure what or if that's the case here as well